tests/testeventtypedb.c

```diff
@@ -301,12 +301,12 @@ test_db (void)
   /* also that trying the wrong type fails */
   assert (
     ! lwes_event_type_db_check_for_type ( db,
-                                        LWES_U_INT_16_TOKEN,
+                                        LWES_TYPE_U_INT_16,
                                         (LWES_SHORT_STRING)"aMetaString",
                                         (LWES_SHORT_STRING)"TypeChecker"));
   assert (
     ! lwes_event_type_db_check_for_type ( db,
-                                          LWES_U_INT_16_TOKEN,
+                                          LWES_TYPE_U_INT_16,
                                           (LWES_SHORT_STRING)"random",
                                           (LWES_SHORT_STRING)"TypeChecker"));
 
```

Comment:
Will type checks for arrays and nullable arrays work?  If so can we add them?

Reply:
Added, around line 203.